docs/Performance.md

```diff
@@ -0,0 +1,242 @@
+# Protobuf Perforamcne
+This benchmark result is tested on workstation with processor of Intel® Xeon® Processor E5-2630 and 32GB RAM
+
+## Parsing performance
+<table>
+<tbody><tr>
+<th rowspan="2"> </th>
+<th colspan="3" rowspan="1">cpp</th>
+<th colspan="3" rowspan="1">java</th>
+<th colspan="3" rowspan="1">python</th>
+</tr>
+<tr>
+<th colspan="1">new(use new operator for creating message instance)</th>
```

Comment:
This isn't very readable when embedded into the table column. Can you add it as a separate list before the table? I should cover C++ and also other languages.

Reply:
Done